app/views/keywords/index.html.slim

```diff
@@ -1,10 +1,26 @@
-p[style="color: green"]
-  = notice
+section.keywords
+  .container
+    .keywords__top
+      img src="#{image_url 'nimblehq_logo.svg'}" alt="nimblehq"
+      .keywords__hero
+        h1 G-Scraper
+        p Scrape data right off Google!
```

Comment:
I think we can turn these into a banner component.
And then we can have classes for `img`, `h1` and `p`

```
.app-banner
  .app-banner__logo
  .app-banner__headline
  .app-banner__about
```

Reply:
Yup split up the code and made it into a component instead 😄 Updated here https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/4de259ac3b8d20a892a061fd18c97a8a63517e55